File: app/src/main/java/com/github/weatherapp/core/CurrentForecast.java
package com.github.weatherapp.core;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class CurrentForecast {
    @SerializedName("name")
    public String city;

    @SerializedName("weather")
    public List<Weather> weatherList;

    @SerializedName("main")
    public WeatherData weatherData;

    @SerializedName("sys")
    public Country country;
}

File: app-kotlin/src/main/kotlin/com/github/weatherapp/core/CurrentForecast.kt
package com.github.weatherapp.core

import com.google.gson.annotations.SerializedName

class CurrentForecast {
    @SerializedName("name")
    var city: String? = null

    @SerializedName("weather")
    var weatherList: List<Weather>? = null

    @SerializedName("main")
    var weatherData: WeatherData? = null

    @SerializedName("sys")
    var country: Country? = null
}
